pom.xml

```diff
@@ -154,6 +154,7 @@
                 <hdp_version>2.6.5.0</hdp_version>
                 <global_hbase_version>1.1.1</global_hbase_version>
                 <global_hbase_guava_version>12.0</global_hbase_guava_version>
+                <global_solr_version>7.4.0</global_solr_version>
```

Comment:
The code changes in this PR will not compile with 6.6.2.  I set all profiles to the new version in case people still want to submit PRs for other dependencies using the old profile.  I created a Jira to clean this up once all components have been upgraded.  In the end I don’t see us having multiple profiles anyways.